core/src/com/unciv/logic/city/CityInfo.kt

```diff
@@ -180,7 +180,15 @@ class CityInfo {
             val amount = getTileResourceAmount(tileInfo) * civInfo.getResourceModifier(resource)
             if (amount > 0) cityResources.add(resource, amount, "Tiles")
         }
+        for (tileInfo in getTiles().filter { it.improvement!= null }) {
+            val tileImprovement = tileInfo.getTileImprovement() ?: continue
+            for (unique in tileImprovement.uniqueObjects)
+                if (unique.placeholderText == "Provides [] []"){
```

Comment:
Flip this if as well :)

Reply:
if (unique.placeholderText == "Provides [] []"){
this one? I'm not sure what you mean sorry :(

Reply:
think i figured it out :)